### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Mzuha**

### CODE REVIEW

File: src/main/java/mate/academy/model/MovieSession.java [hunk 1]
```diff
@@ -15,15 +16,15 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (EugeneSinica) [inline src/main/java/mate/academy/model/MovieSession.java:18]: i am not sure that eager is a good idea, but i don't know
let Sofia resolve my question

**Review by EugeneSinica — APPROVED** [review]

File: .jpb/jpb-settings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Sofasmile) [inline .jpb/jpb-settings.xml:1]: цей файл повинен потрапити в ПР?

File: src/main/java/mate/academy/dao/OrderDao.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    List<Order> getOrderHistory(User user);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/dao/OrderDao.java:10]:
```suggestion
    List<Order> getByUser(User user);
```

File: src/main/java/mate/academy/model/MovieSession.java (hunk 1 above)
Comment (Sofasmile) [inline src/main/java/mate/academy/model/MovieSession.java:18]: for Many-to-One and One-to-One default eager is ok

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**